agent/tool-scripts/postprocess/openvswitch-postprocess

```diff
@@ -367,6 +367,16 @@ while (my $line = <TXT>) {
 		next;
 	}
 
+	#       smc hits:0
+	if ( $line =~ /\s*smc\shits:\s*(\d+)/ and $stats_mode eq "dpdk" ) {
```

Comment:
This code looks just like the above code for `emc hits` but just using `smc` instead of `emc`.  Any way we can apply the DRY principle here?

Reply:
@portante @k-rister perhaps, if we modify:

<pre>
if ( $line =~ /\s*emc\shits:\s*(\d+)/ and $stats_mode eq "dpdk" ) {
	if (exists $pmd_mappings{$pmd_thread_id}) {
			$rate{"openvswitch"}{"PMDthread-emc-hits"}{$pmd_thread_id . $pmd_mappings{$pmd_thread_id}}{$timestamp_ms} = $1;
		} else {
			printf "warning: a emc hits stat for $pmd_thread_id was found, but there was no record of this PMD thread in pmd-rxq-mapping.txt\n";
		}
		next;
</pre>
to be:

<pre>
if ( $line =~ /\s*([es]mc)\shits:\s*(\d+)/ and $stats_mode eq "dpdk" ) {
	if (exists $pmd_mappings{$pmd_thread_id}) {
			$rate{"openvswitch"}{"PMDthread-" . $1 . "-hits"}{$pmd_thread_id . $pmd_mappings{$pmd_thread_id}}{$timestamp_ms} = $2;
		} else {
			printf "warning: a %s hits stat for $pmd_thread_id was found, but there was no record of this PMD thread in pmd-rxq-mapping.txt\n", $1;
		}
		next;
</pre>

And then the section for sac is removed.

Reply:
As is I was just following the flow where every other line has it's own parsing block but I can certainly tweak it.